src/graph.rs

```diff
@@ -170,6 +170,31 @@ impl<T> Graph<T> {
         id
     }
 
+    /// Returns all inserted edges.
+    ///
+    /// ## Example
+    /// ```rust
+    /// use graphlib::{Graph, GraphErr, VertexId};
+    ///
+    /// let mut graph: Graph<usize> = Graph::new();
+    ///
+    /// // Id of vertex that is not place in the graph
+    /// let id = VertexId::random();
+    ///
+    /// let v1 = graph.add_vertex(1);
+    /// let v2 = graph.add_vertex(2);
+    ///
+    /// // Adding an edge is idempotent
+    /// graph.add_edge(&v1, &v2);
+    /// graph.add_edge(&v1, &v2);
+    /// graph.add_edge(&v1, &v2);
+    ///
+    /// assert_eq!(graph.edges().unwrap().len(), 1);
+    /// ```
+    pub fn edges(&self) -> Result<(&Vec<Edge>), GraphErr> {
```

Comment:
The graph api shouldn't expose an `edges` method.

Reply:
I dropped the `pub`. Now its private. Ok?